src/audio/volume/Kconfig

```diff
@@ -8,6 +8,8 @@
 
 if COMP_VOLUME
 
+rsource "Kconfig.hifi"
```

Comment:
Kconfig.simd

Reply:
simd means single instruction multiple data, for example, a 32-bit register, it can load 2 16-bit to perform 2 times multiply with one instruction, no matter hifhi3/4/5, both support same simd, however, the difference is the numbers: i.e. how many multiply unit it have in current HIFI DSP, for example, HIFI3 can perform 3 32-bit multiply in one cycle, HIFI4 can perform 4 times 32-bit multiple in one cycle. hifi is more accurate than simd, we are choosing hifi version with this kconfig.

Reply:
simd is a generic term, we will be using the same method/file for ARM and x86 SIMD so pls use SIMD